src/commands/lambda/flare.ts

```diff
@@ -229,6 +242,62 @@ export class LambdaFlareCommand extends Command {
   }
 }
 
+/**
+ * Obfuscate the environment variables in a Lambda function configuration
+ * @param config
+ */
+export const obfuscateConfig = (config: FunctionConfiguration) => {
+  const environmentVariables = config.Environment?.Variables
+  if (!environmentVariables) {
+    return config
+  }
+
+  const obfuscatedEnvironmentVariables: {[key: string]: string} = {}
+  for (const [key, value] of Object.entries(environmentVariables)) {
+    obfuscatedEnvironmentVariables[key] = value
+    if (!NON_OBFUSCATED_ENV_VARS.has(key)) {
+      obfuscatedEnvironmentVariables[key] = getObfuscation(value)
+    }
+  }
+
+  return {
+    ...config,
+    Environment: {
+      ...config.Environment,
+      Variables: obfuscatedEnvironmentVariables,
+    },
+  }
+}
+
+/**
+ * Obfuscate a string but keep the first two and last four characters
+ * Obfuscate the entire string if it's short
+ * @param original the string to obfuscate
+ * @returns the obfuscated string
+ */
+export const getObfuscation = (original: string) => {
+  // Don't obfuscate booleans
+  if (original.toLowerCase() === 'true' || original.toLowerCase() === 'false') {
+    return original
+  }
+
+  // Dont obfuscate numbers
+  if (!isNaN(Number(original))) {
+    return original
+  }
```

Comment:
I used `Number` instead of `parseInt` because `Number('123abc')` will return `NaN` while `parseInt('123abc')` will return `123`.